test/parallel/test-no-addons-resolution-condition.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+const common = require('../common');
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+const { createRequire } = require('module');
+
+const loadFixture = createRequire(fixtures.path('node_modules'));
+
+if (isMainThread) {
+  [[], ['--no-addons']].map((execArgv) => {
```

Comment:
If the function here is not returning a value, use `forEach()` instead.

Reply:
Oh yea, you're right. I actually wanted to change it to a `for-of` loop. Is that cool too?